domains/certificates/Query.API/API/DataSyncSyncer/Startup.cs

```diff
@@ -1,9 +1,18 @@
+using System;
```

Comment:
Can all in Startup.cs be moved into Program.cs?

Reply:
We have a separate Startup in each component to separate the different services and easily replace them when needed.